stack/corepersistence/collection/src/main/java/org/apache/usergrid/persistence/collection/serialization/impl/MvccEntitySerializationStrategyV3Impl.java

```diff
@@ -458,9 +460,9 @@ public ByteBuffer toByteBuffer( final EntityWrapper wrapper ) {
             final Entity entity = wrapper.entity.get();
             //Convert to internal entity map
             final EntityMap entityMap = EntityMap.fromEntity( entity );
-            final byte[] entityBytes;
+            final String entityString;
             try {
-                entityBytes = MAPPER.writeValueAsBytes( entityMap );
+                entityString = MAPPER.writeValueAsString( entityMap );
```

Comment:
This still seems to use the fieldbuffer byte buffer.  Can we remove this completely and just use a nested JSON object?